flocker/restapi/docs/publicapi.py

```diff
@@ -420,7 +407,13 @@ def run(self):
 
         appContainer = namedAny(self.arguments[0])
 
-        examples_path = FilePath(self.options["examples_path"])
+        # For now we ignore this and just hardcode, due to
+        # https://github.com/sphinx-doc/sphinx/issues/1685
+        # examples_path = FilePath(self.src).preauthChild(
+        #     options["examples_path"]))
```

Comment:
I haven't tested it, but I think ``self.state_machine.get_source(self.lineno)`` will get the right file.